admin/CopyReplication

```diff
@@ -12,35 +12,114 @@ cd "$MB_SERVER_ROOT"
 
 source admin/config.sh
 
-TEMP_DIR=$1 ; shift
+TEMP_DIR="$1" ; shift
 
-# Was a replication packet created?
-shopt -s nullglob
-REPLICATION_FILE=`echo -n "$TEMP_DIR"/replication-*.tar.bz2`
```

Comment:
Instead of enabling it for the whole script, I added a separate loop before the main one where I push all of the new packets into an array.  Does that seem sensible?